WireAuthentication/Sources/WireAuthentication/Components/DetermineAuthMethodComponent.swift

```diff
@@ -54,7 +54,8 @@ class DetermineAuthMethodComponent: Component<DetermineAuthMethodComponentDepend
     @MainActor var determineAuthMethodView: DetermineAuthMethodView {
         DetermineAuthMethodView(
             viewModel: viewModel,
-            builder: loginViaEmailComponent
+            builder: loginViaEmailComponent,
```

Comment:
Should we rename this `builder`?

Reply:
Do you mean, rename it to `loginViaEmailBuilder`? I would say yes.